packages/main/src/components/Form/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { Grid } from '@ui5/webcomponents-react/lib/Grid';
+import React, { FC, forwardRef, ReactElement, ReactNode, ReactNodeArray, Ref, useMemo } from 'react';
+import { CommonProps } from '../../interfaces/CommonProps';
+import { Title } from '@ui5/webcomponents-react/lib/Title';
+import { TitleLevel } from '@ui5/webcomponents-react/lib/TitleLevel';
+import styles from './Form.jss';
+import { createUseStyles } from 'react-jss';
+import { useViewportRange } from '../../../../base/src/hooks/useViewportRange';
```

Comment:
Please use absolute imports. This import might be the reason the build is failing ;)